core/src/cg/group4/view/screen/CollectiblesScreen.java

```diff
@@ -229,33 +225,25 @@ public void display() {
      */
     protected void constructContents() {
         Gdx.app.log(getClass().getSimpleName(), "Construction");
-
-        ArrayList<Collectible> sortedList = cSelectedCollection.sort(cSorter);
-        DecimalFormat format = new DecimalFormat("#.00");
+        final Player player = StandUp.getInstance().getPlayer();
+        DecimalFormat format = new DecimalFormat("0.##");
 
         boolean myCollection = cGroupsBox.getSelectedIndex() == 0
```

Comment:
I'm agree since it it is only used on one spot. I prefer the multiple booleans but I've no objection to your version.